invenio.cfg

```diff
@@ -350,59 +352,96 @@ RDM_NAMESPACES = {
     "cern": "https://greybook.cern.ch/",
 }
 
-# RDM_CUSTOM_FIELDS = [
-#     VocabularyCF(
-#         name="cern:experiment",
-#         vocabulary_id="experiments",
-#         dump_options=True, 
-#         multiple=False,
-#     ),
-#     VocabularyCF(
-#         name="cern:department",
-#         vocabulary_id="departments",
-#         dump_options=True,
-#         multiple=False,
-#     ),
-# ]
+RDM_CUSTOM_FIELDS = [
+    VocabularyCF(
+        name="cern:experiments",
+        vocabulary_id="experiments",
+        dump_options=True,
+        multiple=True,
+    ),
+    VocabularyCF(
+        name="cern:departments",
+        vocabulary_id="departments",
+        dump_options=True,
+        multiple=True,
+    ),
+    VocabularyCF(
+        name="cern:accelerators",
+        vocabulary_id="accelerators",
+        dump_options=True,
+        multiple=True,
+    ),
+    KeywordCF(name="cern:projects", multiple=True),
+    KeywordCF(name="cern:facilities", multiple=True),
+    KeywordCF(name="cern:studies", multiple=True),
+]
 
 THEME_MATHJAX_CDN = (
     "//cdnjs.cloudflare.com/ajax/libs/mathjax/3.2.2/es5/tex-mml-chtml.js"
     "?config=TeX-AMS-MML_HTMLorMML"
 )
 
-# RDM_CUSTOM_FIELDS_UI = [
-#     {
-#         "section": "CERN",
-#         "displaySection": False,
-#         "fields": [
-#             dict(
-#                 field="cern:experiment",
-#                 ui_widget="AutocompleteDropdown",
-#                 display_url="https://greybook.cern.ch/experiment/list",
-#                 props=dict(
-#                     label="Experiment",
-#                     icon="lab",
-#                     placeholder="Select an experiment",
-#                     description="You should fill this field with one of the experiments e.g ATLAS, CMS etc.",
-#                     search=True,
-#                     multiple=False,
-#                     clearable=True,
-#                     autocompleteFrom="/api/vocabularies/experiments",
-#                 )
-#             ),
-#             dict(
-#                 field="cern:department",
-#                 ui_widget="Dropdown",
-#                 display_url="https://scientific-info.cern/archives/history_CERN/internal_organisation/20s",
-#                 props=dict(
-#                     label="Department",
-#                     icon="building",
-#                     description="You should fill this field with one of the departments e.g BE, EN, HR etc.",
-#                     search=False,
-#                     multiple=False,
-#                     clearable=True,
-#                 )
-#             ),
-#         ]
-#     }
-# ]
\ No newline at end of file
+RDM_CUSTOM_FIELDS_UI = [
+    {
+        "section": "CERN",
+        "displaySection": False,
+        "fields": [
+            dict(
+                field="cern:experiments",
+                ui_widget="AutocompleteDropdown",
+                display_url="https://greybook.cern.ch/experiment/list",
```

Comment:
Where is this displayed? 
minor: worth making it a config in case we need to change this?